src/vis/vis.js

```diff
@@ -23,17 +23,74 @@ var INFOWINDOW_TEMPLATE = require('./vis/infowindow-template');
 var WidgetsView = require('cdb/geo/ui/widgets/widgets_view');
 var CartoDBLayerGroupNamed = require('cdb/geo/map/cartodb-layer-group-named');
 var CartoDBLayerGroupAnonymous = require('cdb/geo/map/cartodb-layer-group-anonymous');
-var TimeWidgetView = require('cdb/geo/ui/widgets/time/view');
-var Model = require('cdb/core/model');
+var WidgetViewFactory = require('cdb/geo/ui/widgets/widget_view_factory');
+var ListWidgetView = require('cdb/geo/ui/widgets/list/view');
+var QuantitiesHistogramView = require('cdb/geo/ui/widgets/histogram/quantities-view');
+var TimeHistogramView = require('cdb/geo/ui/widgets/histogram/time-view');
+var CategoryWidgetView = require('cdb/geo/ui/widgets/category/view');
+
+var isTimeHistogramWidget = function(m) {
+  return m.get('type') === 'histogram' && m.get('column_type') === 'date';
+};
 
 /**
- * visulization creation
+ * Visualization creation
  */
 var Vis = View.extend({
 
   initialize: function() {
     _.bindAll(this, 'loadingTiles', 'loadTiles', '_onResize');
 
+    // TODO this should probably be extracted, together with the .load method
```

Comment:
Kind of related to #804, I configure the widget-view-factory here since this `vis.load` is where all things are wired up. cc @alonsogarciapablo @acanimal